pkg/await/apps_deployment.go

```diff
@@ -342,7 +344,8 @@ func (dia *deploymentInitAwaiter) checkAndLogStatus() bool {
 				return false
 			}
 
-			dia.config.logStatus(diag.Info, "✅ Deployment initialization complete")
+			dia.config.logStatus(diag.Info,
+				fmt.Sprintf("%s Deployment initialization complete", cmdutil.EmojiOr("✅", "")))
```

Comment:
```suggestion
				fmt.Sprintf("%sDeployment initialization complete", cmdutil.EmojiOr("✅ ", "")))
```

Same feedback throughout

Reply:
(assuming that works and the format string parser doesn't choke on `%sDeployment`)